### PR TITLE
Event and EventGroup components tests

### PR DESCRIPTION
**Pull request by mhib**

### CODE REVIEW

File: client/test/event/EventTest.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    conference_room: sampleRoom,
+    start_timestamp: 1469397600,
+    end_timestamp: 1469406600,
+    summary: eventSummary
+  };
+  const containerHeight = 30;
+  const unitEventLengthInSeconds = 30 * 60;
+  const timeFormat = 'HH:mm';
+
+  jsdom();
```
Comment (quirk0o) [inline client/test/event/EventTest.jsx]: Jsdom is unnecessary when using `shallow`

File: client/test/event/EventTest.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    expect(wrapper.find('.event-name').text()).to.include(eventSummary);
+    expect(wrapper.find('.event-user').text()).to.include(creator.email);
+    expect(wrapper.find('.event-location').text()).to.include(sampleRoom.title);
+  });
+
+  it('renders correctly with other length', () => {
+    let eventClone = _.cloneDeep(sampleEvent);
+    eventClone.end_timestamp += 60 * 60; // 1 hour
+    eventClone.end.date_time = '2016-07-25T03:30:00.000+02:00';
+    const wrapper = mount(
```
Comment (quirk0o) [inline client/test/event/EventTest.jsx]: Pretty sure you can use `shallow` here as well

File: client/test/event/EventTest.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    expect(wrapper.find('.event-time').text()).to.include('00:30');
+    expect(wrapper.find('.event-name').text()).to.include(eventSummary);
+    expect(wrapper.find('.event-user').text()).to.include(creator.email);
+    expect(wrapper.find('.event-location').text()).to.include(sampleRoom.title);
+  });
+
+  it('renders correctly with other length', () => {
+    let eventClone = _.cloneDeep(sampleEvent);
+    eventClone.end_timestamp += 60 * 60; // 1 hour
+    eventClone.end.date_time = '2016-07-25T03:30:00.000+02:00';
```
Comment (quirk0o) [inline client/test/event/EventTest.jsx]: Why is this necessary?

File: client/test/event/EventTest.jsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import Event from '../../app/components/calendar/event/Event';
+import chai from 'chai';
+import jsdom from 'mocha-jsdom';
+import { _ } from 'lodash';
+
+const expect = chai.expect;
```
Comment (quirk0o) [inline client/test/event/EventTest.jsx]: You can just do

```
import { expect } from 'chai';
```

:P

File: client/test/event/EventTest.jsx (hunk 3 above)
Comment (mhib) [inline client/test/event/EventTest.jsx]: It is not. I just thought that if timestamp changed the date_time should be updated as well.

File: client/test/event/Event.spec.jsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  it('renders correctly with other length', () => {
+    let eventClone = _.cloneDeep(sampleEvent);
+    eventClone.end_timestamp += 60 * 60; // 1 hour
+    eventClone.end.date_time = '2016-07-25T03:30:00.000+02:00';
+    const wrapper = shallow(
+      <Event
+      event={eventClone}
+      containerHeight={containerHeight}
+      unitEventLengthInSeconds={unitEventLengthInSeconds}
+      timeFormat={timeFormat} />
```
Comment (quirk0o) [inline client/test/event/Event.spec.jsx]: This indentation style is really weird :P
Indent props 2 spaces further than opening tag, nei? :P

File: client/test/factories/Event.js [hunk 6]
```diff
@@ -10,6 +10,7 @@ export default new Factory()
   .attr('end_timestamp', ['end_time'], endTime => endTime.getTime() / 1000)
   .attr('start', ['start_time'], startTime => ({ date_time: startTime.toISOString() }))
   .attr('end', ['end_time'], endTime => ({ date_time: endTime.toISOString() }))
+  .attr('summary', 'Conference')
```
Comment (quirk0o) [inline client/test/factories/Event.js]: Summary is not a required event prop

File: client/test/components/calendar/event/Event.spec.jsx [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  it('renders correctly', () => {
+    const event = EventFactory.build();
+    const wrapper = shallow(
+      <Event
+        event={event}
+        containerHeight={containerHeight}
+        unitEventLengthInSeconds={unitEventLengthInSeconds}
+        timeFormat={timeFormat} />
+    );
+    expect(wrapper.find('div')).to.have.lengthOf(5);
```
Comment (madejejej) [inline client/test/components/calendar/event/Event.spec.jsx:21]: I think the first assertion may be an overkill
